README.md

```diff
@@ -107,27 +112,13 @@ To run all queries, cd to `tpcx_bb/` and:
 python benchmark_runner.py --config_file benchmark_runner/benchmark_config.yaml
 ```
 
-By default, this will run each query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.
+By default, this will run each pure Dask query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.
```

Comment:
```suggestion
By default, this will run each Dask query once, and, if BlazingSQL queries are enabled in `benchmark_config.yaml`, each BlazingSQL query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.
```